visidata/canvas.py

```diff
@@ -808,6 +865,13 @@ def deleteSourceRows(self, rows):
 Canvas.addCommand('g'+ENTER, 'dive-visible', 'vs=copy(source); vs.rows=list(rowsWithin(plotterVisibleBox)); vd.push(vs)', 'open sheet of source rows visible on screen')
 Canvas.addCommand('gd', 'delete-visible', 'deleteSourceRows(rowsWithin(plotterVisibleBox))', 'delete rows on source sheet visible on screen')
 
+Canvas.addCommand('', 'add-line-x', 'sheet.gridlines_x += [float(x) for x in input("add line(s) at x = ", value="0.0", defaultLast=True).split()]; sheet.refresh()', 'draw a vertical line at x-values (space-separated)')
```

Comment:
I could see bindings being `gx` and `gy` to add a gridline, and `zx`/`zy` to remove them (and `gzx`/`gzy` to clear them).  Maybe swapping `gx`/`gy` and current `x`/`y`.  What do you think about this?

Reply:
Those bindings all sound reasonable to me. I don't use current `x` and `y` so I don't have an opinion on those.